.github/ISSUE_TEMPLATE/1_bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: 'Bug report'
+about: 'Report a bug in miranum'
+title:
+labels: 'type: bug :bug:'
+assignees: dominikhorn93
```

Comment:
I think it's important to respond to new issues within a couple of days. Should we only assign all issues to dominik? We could add more maintainers to the list or setup a bot that posts messages to a ms teams channel